src/java.base/share/classes/java/lang/invoke/MethodHandles.java

```diff
@@ -4900,7 +4890,7 @@ private static MethodHandle identityOrVoid(Class<?> type) {
      * @param type the type of the desired method handle
      * @return a constant method handle of the given type, which returns a default value of the given return type
      * @throws NullPointerException if the argument is null
-     * @see MethodHandles#zero
+     * @see MethodHandles#primitiveZero
```

Comment:
`MethodHandles​.primitiveZero(…)` is a private internal API.
```suggestion
     * @see MethodHandles#zero
```

Reply:
There's a general rfe for checking inaccessible links. Don't recall why thisv was even changed, most likely ide being too smart.